addon/components/ember-popper-base.js

```diff
@@ -104,41 +135,49 @@ export default class EmberPopperBase extends Component {
 
   // ================== PRIVATE IMPLEMENTATION DETAILS ==================
 
-  _addPopper() {
-    const target = this.get('_popperTarget');
-    const element = this.get('_popperElement');
-    const options = this.get('_options');
-
-    this._popper = new Popper(target, element, options);
-  }
-
-  /**
-   * The element to be positioned by the Popper.
-   */
-  @computed('_renderInPlace')
-  _popperElement() {
-    assert('_popperElement must only be used after the component is rendered', !!this.element);
-
-    const renderInPlace = this.get('_renderInPlace');
+  _updatePopper() {
+    const popperElement = this._popperElement;
+    const popperTarget = this._getPopperTarget();
+    const eventsEnabled = this.get('eventsEnabled');
+    const modifiers = this.get('modifiers');
+    const placement = this.get('placement');
 
-    // If not rendering in place, return the wrapper around yielded elements. See template for details.
-    return renderInPlace ? this.element : document.querySelector(`.popper-${this.elementId}`);
+    // Compare against previous values to see if anything has changed
+    const didChange = popperElement !== this._popperElement
```

Comment:
Whoops lol, yeah that was basically what that was :P changed a bit though, let me fix it